server/server.js

```diff
@@ -1,24 +1,22 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const expressSession = require('express-session');
+const PGSession = require('connect-pg-simple')(expressSession);
 const passport = require('passport');
 const Auth0Strategy = require('passport-auth0');
+const path = require("path");
 const dotenv = require('dotenv');
-const db = require('./db/db');
 const apiRouter = require('./api');
+const pgPool = require('./db/db');
```

Comment:
we should keep this labeled as `db` for the variable reference since it is the DB file. If you're trying to access pgPool explicitly then you should access it explicitly like `db.pgPool`

```suggestion
const db = require('./db/db');
```